python/stepfunctions/stepfunctions/stepfunctions_stack.py

```diff
@@ -9,50 +9,77 @@ class JobPollerStack(core.Stack):
     def __init__(self, app: core.App, id: str, **kwargs) -> None:
         super().__init__(app, id, **kwargs)
 
-        submit_job_activity = sfn.Activity(
```

Comment:
These don't align with our syntactic python conventions. Please revert to `xx = xx`

Reply:
reverted to xx = xx.